composer.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "fusesource/stomp-php",
     "description": "stomp support for PHP",
+    "abandoned": true,
```

Comment:
The abandon flag seems to be only available for the satis configuration. I abandoned the Package in packagist and named stomp-php/stomp-php as the follow up